setpass/api.py

```diff
@@ -166,5 +168,46 @@ def add(user_id):
     return Response(response=user.token, status=200)
 
 
+@wsgi.app.route('/reset', methods=['GET'])
+def view_reset_form():
+    return render_template('reset_form.html')
+
+
+@wsgi.app.route('/reset', methods=['POST'])
+def reset_password():
```

Comment:
Thanks for getting back to me. I think you don't have to change the name. It is probably a convention to only call that `reset` instead of, "delete and create a new one". 

Yeah, I thought it should be "update" but based on you said, it makes sense and that looks good to me 👍